CRM/Core/Permission.php

```diff
@@ -278,7 +278,7 @@ public static function ufGroupValid($gid, $type = CRM_Core_Permission::VIEW) {
     }
 
     $groups = self::ufGroup($type);
-    return !empty($groups) && in_array($gid, $groups) ? TRUE : FALSE;
+    return !empty($groups) && in_array($gid, $groups);
```

Comment:
Off the top of my head, I'm not sure if `&&` or `?:` has higher precedence, but the old snippet is equally redundant under either interpretation. 🙃 

(Aside: it turns out that the `&&` is evaluated first, which does seem like the more intuitive precedence...)